hilti/toolchain/include/ast/node.h

```diff
@@ -127,7 +127,7 @@ inline std::string to_string(const PropertyValue& v) {
         auto operator()(int i) { return util::fmt("%d", i); }
         auto operator()(int64_t i) { return util::fmt("%" PRId64, i); }
         auto operator()(const std::string& s) { return util::escapeUTF8(s); }
-        auto operator()(const ID& id) { return id.str(); }
+        auto operator()(const ID& id) { return std::string(id.str()); }
```

Comment:
This seems unnecessary,

```suggestion
        auto operator()(const ID& id) { return id.str(); }
```

If some compiler complains about the differing return values from the visitors, maybe it would be clearer to instead make the return type explicit (e.g., `std::string operator()(const ID& id) { return id.str(); }` for all visitors).

Reply:
This was just a left-over that's no longer needed.